pix-editor/app/controllers/authenticated/static-courses/static-course/edit.js

```diff
@@ -8,17 +8,23 @@ export default class EditStaticCourseController extends Controller {
   @service notifications;
 
   get challengeIdsAsStringWithBreakLines() {
-    return this.model.sortedChallengeSummaries.toArray()
+    return this.model.staticCourse.sortedChallengeSummaries.toArray()
       .map((challengeSummary) => challengeSummary.id)
       .join('\n');
   }
 
+  get tagIds() {
+    return this.model.staticCourse.tags.toArray()
+      .sort((tagA, tagB) => tagA.label.toLowerCase() > tagB.label.toLowerCase())
+      .map(({ id }) => id);
+  }
+
   @action
   async editStaticCourse(formData) {
     try {
-      await this.model.save({ adapterOptions: { ...formData, action: 'update' } });
+      await this.model.staticCourse.save({ adapterOptions: { ...formData, action: 'update' } });
```

Comment:
**suggestion:** C'est ici que l'on devrait peut-être assigner les valeurs de `formData` dans `this.model.staticCourse`, plutôt que de les envoyer dans `adapterOptions`, ce serait plus proche du *ember way of doing things*:

```suggestion
      Object.assign(this.model.staticCourse, formData);
      await this.model.staticCourse.save({ adapterOptions: { action: 'update' } });
```

Reply:
non car les attributs du modèle static course ne correspondent pas exactement à la commande envoyée côté API. Après c'est surtout vis à vis des relations qu'on prend un raccourci. J'imagine qu'on pourrait ajouter les relationships proprement et envoyer la data.. On va essayer